pkg/kubelet/eviction/helpers.go

```diff
@@ -699,13 +733,13 @@ func disk(stats statsFunc, fsStatsToMeasure []fsStatsType, diskResource v1.Resou
 
 // rankMemoryPressure orders the input pods for eviction in response to memory pressure.
 func rankMemoryPressure(pods []*v1.Pod, stats statsFunc) {
-	orderedBy(qosComparator, memory(stats)).Sort(pods)
+	orderedBy(exceedMemoryRequests(stats), priority, memory(stats)).Sort(pods)
```

Comment:
Among all abusive users, we sorted them based on the priority first, then memory usage, not (usage - request)?

Reply:
This function is already (usage - request), but is named memory for readability.

Reply:
renamed

Reply:
decided to change it back.  I think it is less confusing the original way